Endpoints/app.js

```diff
@@ -6,28 +6,27 @@ const {
   getArticlesByArticleID,
   getAllArticles,
   getCommentsById,
-} = require("../Controllers/get.controllers");
+  postCommentToArticleId,
+} = require("../Controllers/news.controllers");
+const {
+  endpointErrors,
+  psqlErrors,
+  customErrors,
+  serverErrors,
+} = require("../Errors/errorHandler");
+
+app.use(express.json());
```

Comment:
Good to see you only now adding this middleware when you actually need it, well done.